src/main/java/de/tum/in/www1/artemis/service/user/UserService.java

```diff
@@ -467,10 +473,11 @@ protected void anonymizeUser(User user) {
         final Set<String> originalGroups = user.getGroups();
         final String randomPassword = RandomUtil.generatePassword();
         final String userImageString = user.getImageUrl();
+        final String anonymizedLogin = lowerCase(RandomUtil.generateRandomAlphanumericString(), Locale.ENGLISH);
```

Comment:
do we really need to import this lowerCase function?
I guess the `"......".toLowerCase()` java standard library function would suffice here